src/modules/FunModule.cpp

```diff
@@ -70,31 +70,36 @@ FunModule::FunModule(UmikoBot* client) : Module("funutil", true), m_memeChannel(
 		{
 
 		QStringList args = message.content().split(' ');
-		
-		double max;
-		double min;
 
 		if (args.size() != 3)
 		{
 			client.createMessage(message.channelId(), "**Wrong Usage of Command!** ");
 			return;
 		}
 
+		double min = args.at(1).toDouble();
+		double max = args.at(2).toDouble();
+
 		QRegExp re("[+-]?\\d*\\.?\\d+");
 		if (!re.exactMatch(args.at(1)) || !re.exactMatch(args.at(2)))
 		{
 			client.createMessage(message.channelId(), "**You must roll with numbers!**");
 			return;
 		}
 
+		//Max and min in 32-bit is 2147483647 and -2147483647
+		if (max >= 2147483647 || min >= 2147483647 || max <= -2147483647 || min <= -2147483647)
+		{
+			client.createMessage(message.channelId(), "**You can't roll that number!**");
```

Comment:
This should be more descriptive, *why* can't they roll that number...

Reply:
Will I tell them that it's beyond 32-bit limit? I personally don't like to tell them this 32-bit thingy.